libs/tree-sitter-wing/grammar.js

```diff
@@ -370,6 +371,7 @@ module.exports = grammar({
       prec.right(PREC.OPTIONAL_TEST, seq($.expression, "?")),
 
     compiler_dbg_panic: ($) => "😱",
```

Comment:
As we have more of these, I am wondering if we should put these under some env flag (`DEBUG=1`)...

Reply:
Sounds like a sensible idea, I'll leave it as a future improvement